packages/performance/src/utils/attributes_utils.ts

```diff
@@ -44,12 +44,12 @@ const enum EffectiveConnectionType {
  *
  * ref: https://developer.mozilla.org/en-US/docs/Web/API/NetworkInformation
  */
-interface NetworkInformation {
+interface NetworkInformationWithEffectiveType extends NetworkInformation {
```

Comment:
Just curious: what made these types to rename to `NetworkInformationWithEffectiveType`? Could this be breaking?

Reply:
I don't think so, it's not publicly exposed and is only used here. I will add a comment about what the deal is:

```
  // `effectiveType` is an experimental property and not included in
  // TypeScript's typings for the native NetworkInformation interface
  ```